apps/hash-api/src/graphql/resolvers/knowledge/page/update-page-actions.ts

```diff
@@ -158,7 +152,9 @@ export const handleCreateNewEntity = async (params: {
       throw new UserInputError(`action ${params.index}: ${error}`);
     }
     throw new Error(
-      `createEntity: Could not create new entity: ${JSON.stringify(error)}`,
+      `createEntity: Could not create new entity: ${JSON.stringify(
+        error,
+      )}, trying to create ${JSON.stringify(params)}`,
```

Comment:
I added this for debugging purposes, but it turned out to be quite informative. Can remove if we don't want this in the error.